app/src/componentLibrary/checkbox/checkbox.scss

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.light-container {
+  display: inline-flex;
+  align-items: center;
+  padding-left: 24px;
+  color: $COLOR--almost-black;
+  font-family: $FONT-ROBOTO-REGULAR;
+  font-size: 13px;
+  line-height: 20px;
+  .disabled {
+    opacity: 0.3;
+    cursor: default;
+  }
+}
+
+.input {
+  display: none;
+}
+
+.checkbox {
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  border: 1px solid $COLOR--e-200;
+  border-radius: 3px;
+  margin-left: -24px;
+  box-sizing: border-box;
+  cursor: pointer;
+}
+
+.children-container {
+  cursor: pointer;
+}
+
+.input:checked + .light {
```

Comment:
It would be nice to organize styles in the same way as in [`dropdown`](https://github.com/reportportal/service-ui/pull/3090/files#diff-f89add43efa0f0968bcaf30c2c8056c71f91dd598291516fb656f38d1ddd46d2R138) implementation using SCSS mixins.
It will be easier to add new variants and reduce code duplication.